### PR TITLE
constraint docs fix for spec.artifact

### PR DESCRIPTION
**Pull request by benhealey-rafay**

### CODE REVIEW

File: docs/resources/opa_constraint.md [hunk 1]
```diff
@@ -21,13 +21,14 @@ resource "rafay_opa_constraint" "tfdemoopaconstraint1" {
     project = "terraform"
   }
   spec {
+    template_name = "request-limit-ratio-template"
     artifact {
+      type = "Yaml"
       artifact {
         paths {
           name = "file://artifacts/one/request-limit-ratio-constraint.yaml"
```
Comment (hardik-rafay) [inline docs/resources/opa_constraint.md:29]: Can you replace "one" with "request-limit-ratio" (name of the constarint) int the file path?

File: examples/resources/rafay_opa_constraint/resource.tf [hunk 2]
```diff
@@ -5,13 +5,14 @@ resource "rafay_opa_constraint" "tfdemoopaconstraint1" {
     project = "tfdemoproject1"
   }
   spec {
+    template_name = "one"
     artifact {
+      type = "Yaml"
       artifact {
         paths {
           name = "file://artifacts/one/request-limit-ratio-constraint.yaml"
```
Comment (hardik-rafay) [inline examples/resources/rafay_opa_constraint/resource.tf:13]: same as above.

**Review by hardik-rafay — CHANGES_REQUESTED** [review]

File: examples/resources/rafay_opa_constraint/resource.tf (hunk 2 above)
Comment (benhealey-rafay) [reply examples/resources/rafay_opa_constraint/resource.tf]: done

File: docs/resources/opa_constraint.md (hunk 1 above)
Comment (benhealey-rafay) [reply docs/resources/opa_constraint.md]: done

File: examples/resources/rafay_opa_constraint/resource.tf [hunk 3]
```diff
@@ -5,13 +5,14 @@ resource "rafay_opa_constraint" "tfdemoopaconstraint1" {
     project = "tfdemoproject1"
   }
   spec {
+    template_name = "one"
```
Comment (hardik-rafay) [inline examples/resources/rafay_opa_constraint/resource.tf:8]: template_name = "request-limit-ratio"

**Review by hardik-rafay — CHANGES_REQUESTED** [review]

File: examples/resources/rafay_opa_constraint/resource.tf (hunk 3 above)
Comment (benhealey-rafay) [reply examples/resources/rafay_opa_constraint/resource.tf]: fixed

**Review by hardik-rafay — APPROVED** [review]

**Pull request merged**